lte/gateway/fabfile.py

```diff
@@ -537,7 +535,7 @@ def integ_test_containerized(
 def _start_gateway_containerized():
     """ Starts the containerized AGW """
     with cd(AGW_PYTHON_ROOT):
-        run('make buildenv')
+        run('make buildenv && source ${PYTHON_BUILD}/bin/activate && sudo env PATH=${PATH} snowflake --make-snowflake && cat /etc/snowflake')
```

Comment:
`make buildenv &&  sudo env PATH=${PYTHON_BUILD}/bin:${PATH} snowflake --make-snowflake` would also work.

Reply:
Just out of curiosity, why do we need the `cat /etc/snowflake` here?

Reply:
Afaik, this PR is still very much in working state and the `cat` is just for debugging.